modules/javafx.fxml/src/main/java/javafx/fxml/FXMLLoader.java

```diff
@@ -2321,6 +2321,14 @@ public boolean equals(Object obj) {
         return false;
     }
 
+    @Override
+    public int hashCode() {
+        if (location == null) {
+            return 0;
+        }
+        return location.hashCode();
```

Comment:
Shouldn't this be: `location.toExternalForm().hashCode()`?

Reply:
good point.  will fix.